index.css

```diff
@@ -7,59 +7,82 @@ DARK: #44354A
 COLOR: #9E4770
 */
 
+
+
 html, body {
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-    font-size: 10px;
     background: #201A23;
-    font-family: 'Poppins';
+    font-family: 'Poppins', sans-serif; 
     color: #FBFBFB;
 } 
 
 .salesboard {
-    position: fixed;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    right: 0;
-    margin: auto;
-    text-align: center;
-    width: 50vw;
-
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+   text-align: center;
+   max-width: 940px;
+   margin: 0 auto;
+   padding: 0 15rem;
 }
 
-.salesboard-title {
-    font-size: 4.8rem;
-    margin-bottom: 2rem;
+img {
+    border: 10px solid #44354A;
+    border-radius: 50%;
+    width: 220px;
+    margin-bottom: 25px;
 }
 
-.salesboard-picture {
-    border: 1rem solid #44354A;
-    border-radius: 14.5rem;
-    width: 21.9rem;
-    margin-bottom: 4.5rem;
+button {
+    background: #9E4770;
+    height: 80px;
+    width: 190px;
+    font-size: 45px;
+    border: 0;
+    border-radius: 10px;
+    cursor: pointer;
 }
 
-#btn-productA,
-#btn-productB {
-    width: 20rem;
-    height: 8rem;
-    background: #9E4770;
-    border-radius: 1rem;
+button:active {
+    scale: 0.98;
 }
 
```

Comment:
You can make the h1 slightly bigger ras per the design so it stands out a bit more